inference-runtime/runtime_service/src/paddle/paddle_wrapper.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2020 Intel Corporation
+//
+
+#include <map>
+#include <string>
+#include <vector>
+#include <algorithm>
+#include <syslog.h>
+#include "paddle_wrapper.hpp"
+
+std::string find_param(std::string &model_file) 
+{
+    return model_file.substr(0, model_file.length()-7)+"pdiparams";
+}
+
+PaddleWrapper::PaddleWrapper(const std::string& modelPath):pModel(modelPath) {
+    std::string pParam = find_param(pModel);
+    config.SetModel(pModel, pParam);
+    
+    //config.EnableTensorRtEngine(1 << 30, FLAGS_batch_size, 10, PrecisionType::kFloat32, false, false);
+    predictor = paddle_infer::CreatePredictor(config);
+}
+
+void PaddleWrapper::writeInputData(const cv::Mat& image, uint32_t batchIndex){
+    // tensor_input = torch::from_blob(
+    //                     image.data, {1, image.cols, image.rows, image.channels()});
+    // tensor_input = tensor_input.permute({0, 3, 1, 2});
+    auto input_names = predictor->GetInputNames();
+    tensor_input = predictor->GetInputHandle(input_names[0]);
```

Comment:
Assume the model has only one input and one output. How to support the models with multi-inputs and multi-outputs?